src/memex/search/query.py

```diff
@@ -200,6 +200,19 @@ def __call__(self, params):
         return {'terms': {'user': users}}
 
 
+class DeletedFilter(object):
+
+    """
+    A filter that only returns non-deleted documents.
+
+    Documents are not getting deleted from the index, they only get marked as
+    deleted.
+    """
+
+    def __call__(self, _):
+        return {"bool": {"must_not": {"exists": {"field": "deleted"}}}}
```

Comment:
I see the `missing` filter has been deprecated in more recent Elasticsearch versions. 👍